library/src/main/java/com/bumptech/glide/load/resource/drawable/DrawableResource.java

```diff
@@ -39,8 +39,8 @@ public final T get() {
   public void initialize() {
     if (drawable instanceof BitmapDrawable) {
       ((BitmapDrawable) drawable).getBitmap().prepareToDraw();
-    } else if (drawable instanceof GifDrawable) {
-      ((GifDrawable) drawable).getFirstFrame().prepareToDraw();
+//    } else if (drawable instanceof GifDrawable) {
```

Comment:
forgot?

if you need it how about introducing an interface (so there's only need for BitmapDrawable and that interface for instanceof)